src/main/kotlin/no/digdir/catalog_comments_service/service/CommentService.kt

```diff
@@ -41,4 +42,16 @@ class CommentService (private val commentDAO: CommentDAO, private val userDAO: U
     fun getCommentsByOrgNumberAndTopicId(orgNumber: String, topicId: String): List<Comment> =
         commentDAO.findCommentsByOrgNumberAndTopicId(orgNumber, topicId)
             .map { it.toDTO(userDAO.findById(it.user)) }
+
+    fun getCommentDBO(id: String): CommentDBO? =
+        commentDAO.findByIdOrNull(id)
+
+    fun updateComment(commentId: String, obj: Comment, userId: String): Comment? =
+        commentDAO.findByIdOrNull(commentId)
+            ?.copy(
+                comment = obj.comment ?: ""
```

Comment:
Om det blir gjort et forsøk på å oppdatere med en null-verdi som `comment` så hadde det kanskje vært på plass med BadRequest?

Reply:
```suggestion
                comment = obj.comment ?: throw ResponseStatusException(HttpStatus.BAD_REQUEST)
```
Noe i den duren i så fall, er dog usikker på om det er lov å kaste en exception derfra :P